.mailmap

```diff
@@ -1537,3 +1537,8 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Øyvind Jensen <jensen.oyvind@gmail.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
+Jay Patankar <patankarjays@gmail.com>
```

Comment:
you need to only add your name and email in the `.mailmap` file and then run the following command `python bin/mailmap_check.py` or read [here](https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file)

Reply:
@1e9abhi1e10  I added my name and ran the python bin/mailmap_check.py script . Still I am getting following error:
`Run bin/mailmap_check.py
This author is in the AUTHORS file but not .mailmap:
Charles Harris <erdos4d@gmail.com>
This author is in the AUTHORS file but not .mailmap:
Tejaswini Sanapathi <sastejaswini2002@gmail.com>
This author is in the AUTHORS file but not .mailmap:
Devansh <be19b002@smail.iitm.ac.in>
This author is in the AUTHORS file but not .mailmap:
Aaron Gokaslan <aaronGokaslan@gmail.com>

For instructions on updating the .mailmap file see:
https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file

The following authors will be added to the AUTHORS file at the time of
the next SymPy release.

	Jay Patankar <patankarjays@gmail.com>
Error: Process completed with exit code 1.`

What should be done? Also is there any way of running a particular test once I commit my changes as it is unnecessarily taking a lot of time everytime to execute all the tests  ....Can I rerun only a specific test keeping the pass status of all previous tests intact?